Documentation/ceph-filesystem.md

```diff
@@ -9,7 +9,7 @@ indent: true
 
 A shared filesystem can be mounted with read/write permission from multiple pods. This may be useful for applications which can be clustered using a shared filesystem.
 
-This example runs a shared filesystem for the [kube-registry](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/registry).
+This example runs a shared filesystem for the [kube-registry](https://github.com/kubernetes/kubernetes/tree/release-1.9/cluster/addons/registry).
```

Comment:
This is a really old example on an old K8s version. Is there not a newer version where the example is available? Even better would be to update the example, but that's a bigger work item.

Reply:
Hi, @travisn it seems like it's been removed from K8s. I've notice that it's been set as K8s version 1.9 in current [rook ceph filesystem doc](https://github.com/rook/rook/blob/master/Documentation/ceph-filesystem.md) line 247.

Or maybe there's a better `ReadWriteMany` applies to this document example rather than registry.

---------
# Shared Filesystem

A shared filesystem can be mounted with read/write permission from multiple pods. This may be useful for applications which can be clustered using a shared filesystem.
...
...
## Consume the Shared Filesystem: Toolbox

Once you have pushed an image to the registry (see the [instructions](https://github.com/kubernetes/kubernetes/tree/release-1.9/cluster/addons/registry) to expose and use the kube-registry), verify that kube-registry is using the filesystem that was configured above by mounting the shared filesystem in the toolbox pod. See the [Direct Filesystem](direct-tools.md#shared-filesystem-tools) topic for more details.